model.py

```diff
@@ -0,0 +1,137 @@
+"""Models for Markov chains Tweet generator app"""
+
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import os
+
+db = SQLAlchemy()
+
+
+class User(db.Model):
+    """A user"""
+
+    __tablename__ = 'users'
+
+    user_id = db.Column(db.Integer,
```

Comment:
if this is the id for this table, its better to use just "id". So instead of having to say `user.user_id` or `ug_tweet.ug_tweet_id` which can get kinda wordy, it could just be ug_tweet.id. Readability is the way to go 😄

Reply:
This makes a lot of sense! I tried switching the naming to be just 'id' but kept getting an error. For now, I'll leave it as is, but keep that convention in mind!